@blink1073 - I was getting a typescript build error, and even after I tweaked the example code, I'm still getting:

```
src/index.ts(85,7): error TS2322: Type 'JavascriptRenderer' is not assignable to type 'IRenderer<Widget>'.
  Types of property 'render' are incompatible.
    Type '(mimetype: string, data: string) => Widget' is not assignable to type '(mimetype: string, data: string) => Widget'.
      Type 'Widget' is not assignable to type 'Widget'.
        Types of property 'layout' are incompatible.
          Type 'Layout' is not assignable to type 'Layout'.
            Property 'initialize' is protected but type 'Layout' is not a class derived from 'Layout'.
src/index.ts(89,47): error TS2345: Argument of type 'RenderMime<Widget>' is not assignable to parameter of type 'RenderMime<Widget>'.
  Types have separate declarations of a private property '_renderers'.
```

This is on 

```
$ node --version
v4.2.4
```

--- a/example/package.json
+++ b/example/package.json
@@ -3,6 +3,7 @@
   "name": "jupyter-js-notebook-example",
   "dependencies": {
     "jupyter-js-notebook": "file:..",
+    "jupyter-js-ui": "^0.5.1",
     "phosphor-commandpalette": "^0.2.0",
     "phosphor-keymap": "^0.7.0",
     "phosphor-splitpanel": "^1.0.0-rc.1",

